src/sankeyDiagram.ts

```diff
@@ -230,6 +235,7 @@ export class SankeyDiagram implements IVisual {
     private curvatureOfLinks: number = 0.5;
 
     private static NodeAndBackwardLinkDistance = 5;
+    private static DistanceBetweenLinks = 3;
```

Comment:
Type of _**DistanceBetweenLinks**_  variable is absent

Reply:
fixed 
https://github.com/microsoft/powerbi-visuals-sankey/pull/161/commits/0f90ad9016883a0d20c0bf7a137e62b2f99357cf